src/schemas/clearance.schema.ts

```diff
@@ -0,0 +1,13 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+
+export type ClearanceDocument = Clearance & Document;
+
+@Schema()
+export class Clearance {
```

Comment:
you may want to add other fields like academic year if there is separate clearance per AY.